lib/queue-job/job-proxy.js

```diff
@@ -16,7 +16,6 @@ function jobProxy() {
   function processMessage(message, callback) {
 
     try {
-      var data = message && message.data;
```

Comment:
@rozele - is it possible to add validation (make sure message has file / args / name?